gnosis/safe-react
# Modal buttons receive flag to require a selected contract method

## What it solves
Resolves #2313

## How this PR fixes it
In the Contract Intereaction, a prop is passed to the Modal buttons to require a selected method in order to enable the "Review" button

## How to test it
On Contract Interaction, toggle the _Use custom data (hex encoded)_ fill the data
ex:
```
Recipient:
0x49d4450977E2c95362C13D3a31a09311E0Ea26A6
Value:
0
Data:
0x42f4579000000000000000000000000049d4450977e2c95362c13d3a31a09311e0ea26a6
```

And the "Review" button should become enabled

## Screenshots
<img width="497" alt="Screen Shot 2021-11-10 at 23 28 49" src="https://user-images.githubusercontent.com/32431609/141204230-92be90da-17e7-45bf-8316-638178d25cde.png">
